app/src/main/java/com/sunasterisk/weather/data/source/remote/fetchjson/GetJsonFromUrl.kt

```diff
@@ -0,0 +1,24 @@
+package com.sunasterisk.weather.data.source.remote.fetchjson
+
+import android.os.AsyncTask
+import com.sunasterisk.weather.data.model.entity.WeatherStatistics
+import com.sunasterisk.weather.data.source.remote.OnFetchDataJsonListener
+
+class GetJsonFromUrl constructor(private val listener: OnFetchDataJsonListener<WeatherStatistics>) :
+    AsyncTask<String, Void, String>() {
```

Comment:
nếu không private constructor thì bỏ từ khóa constructor đi 
```suggestion
class GetJsonFromUrl(
    private val listener: OnFetchDataJsonListener<WeatherStatistics>
) : AsyncTask<String, Void, String>() {
```